lib/config/sentry.rb

```diff
@@ -31,9 +31,9 @@ module AutoHCK
     # of transactions for performance monitoring.
     config.traces_sample_rate = 1.0
 
-    unless ENV['SENTRY_TRANSPORT_SSL_CA'].nil?
-      if File.exist?(ENV['SENTRY_TRANSPORT_SSL_CA'])
-        config.transport.ssl_ca_file = ENV['SENTRY_TRANSPORT_SSL_CA']
+    unless ENV.fetch('SENTRY_TRANSPORT_SSL_CA', nil).nil?
```

Comment:
`ENV.key?` can be used if you just want to check its existence. I prefer to store the result of `ENV.fetch('SENTRY_TRANSPORT_SSL_CA', nil)` to a variable and use it for both of the existence check and later use in Sentry configuration though.

Reply:
Moved to https://github.com/HCK-CI/AutoHCK/pull/160